bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -438,7 +438,7 @@ async def _edit_nomination_reason(
             json={"actor": actor.id, "reason": reason}
         )
         await self.refresh_cache()  # Update cache
-        await ctx.send(":white_check_mark: Successfully updated nomination reason.")
+        await ctx.send(f":white_check_mark: Successfully updated nomination reason for {actor.mention}")
```

Comment:
Isn't `target` the person being nominated here, with `actor` being the person nominating?

Reply:
Agh, skimmed through and I thought actor would be the person being nominated. Fixed it.